packages/core/src/components/input/input.tsx

```diff
@@ -144,7 +152,7 @@ export class AtomInput {
     return (
       <Host>
         <ion-input
-          ref={(el) => (this.inputEl = el as HTMLIonInputElement)}
+          ref={(el) => (this.inputEl = el)}
```

Comment:
Type assertion removed for sonar-lint resolution
![image](https://github.com/juntossomosmais/atomium/assets/31256653/b281598d-be67-4762-a274-d65bb9cfca2d)